tests/scripts/thread-cert/v1_2_LowPower_5_3_01_SSEDAttachment.py

```diff
@@ -0,0 +1,150 @@
+        self.nodes[SSED_1].start()
+        self.simulator.go(5)
+        self.assertEqual(self.nodes[SSED_1].get_state(), 'child')
+
+        self.simulator.go(5)
+
+        self.collect_rloc16s()
+        self.collect_rlocs()
+
+        # Verify connectivity by sending an ICMP Echo Request from the Router to SSED_1 mesh-local address via the DUT
+        # Set ping timeout as two CSL periods.
+        timeout = 2 * consts.CSL_DEFAULT_PERIOD * consts.US_PER_TEN_SYMBOLS / 1000000
+        self.assertTrue(self.nodes[ROUTER].ping(self.nodes[SSED_1].get_ip6_address(ADDRESS_TYPE.RLOC),
+                                                timeout=timeout))
+
+        # Verify fragmented CSL transmission
+        self.assertTrue(self.nodes[ROUTER].ping(self.nodes[SSED_1].get_ip6_address(ADDRESS_TYPE.RLOC),
+                                                size=128,
+                                                timeout=timeout))
+
+    def verify(self, pv):
+        pkts = pv.pkts
+        pv.summary.show()
+        LEADER = pv.vars['LEADER']
+        ROUTER = pv.vars['ROUTER']
+        SSED_1 = pv.vars['SSED_1']
+
+        # Step 1: ensure the Leader is sending MLE Advertisements
+        pkts.filter_wpan_src64(LEADER) \
+            .filter_mle_cmd(consts.MLE_ADVERTISEMENT) \
+            .must_next()
+
+        # Step 2: Verify that the frame version is IEEE Std 802.15.4-2015 of the MLE Child Update Response from the
+        # DUT to SSED_1
+        pkts.filter_wpan_src64(LEADER) \
+            .filter_wpan_dst64(SSED_1) \
+            .filter_mle_cmd(consts.MLE_CHILD_UPDATE_RESPONSE) \
+            .filter_wpan_version(consts.MAC_FRAME_VERSION_2015) \
+            .must_next()
+
+        # Step 3:
+        # Verify that SSED_1 does not send a MAC Data Request prior to receiving the ICMP Echo Request from the Leader.
+        # Verify that the frame version is IEEE Std 802.15.4-2015 of the ICMP Echo Request sent to SSED_1 from the DUT
+        # Verify that the Router successfully receives an Echo Response from SSED_1
+        ssed_rloc16 = pv.vars['SSED_1_RLOC16']
+        ssed_rloc = pv.vars['SSED_1_RLOC']
+        leader_rloc16 = pv.vars['LEADER_RLOC16']
+        router_rloc16 = pv.vars['ROUTER_RLOC16']
+        router_rloc = pv.vars['ROUTER_RLOC']
+        pkts.filter_wpan_src64(LEADER) \
+            .filter('wpan.dst16 == {rloc16}', rloc16 = ssed_rloc16) \
+            .filter_ipv6_dst(ssed_rloc) \
+            .filter_ping_request() \
+            .filter_wpan_version(consts.MAC_FRAME_VERSION_2015) \
+            .must_next()
+        idx1 = pkts.index
+        pkts.filter_wpan_src64(SSED_1) \
+            .filter('wpan.dst16 == {rloc16}', rloc16 = leader_rloc16) \
+            .filter_ipv6_dst(router_rloc) \
+            .filter_ping_reply() \
+            .filter_wpan_version(consts.MAC_FRAME_VERSION_2015) \
+            .filter_wpan_ie_present() \
+            .filter_wpan_has_csl_ie() \
```

Comment:
What about use these conditions in `must_verify` instead of as filters. 

In general, I think we should use just enough filters to find the exact packet, and then use `must_verify` to verify that packet.